### PR TITLE
Refactor type function

### PR DESCRIPTION
**Pull request by Iz3as**
Toot-toot McBumbersnazzle

### CODE REVIEW

File: screenwriting/screenwriting.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        time.sleep(dur)
+    sys.stdout.write("\n")
+    sys.stdout.flush()
+
+
+def laugh(txt):
+    for letter in txt:
+        sys.stdout.write(letter)
+        sys.stdout.flush()
+        time.sleep(0)
```
Comment (Iz3as) [inline screenwriting/screenwriting.py]: `time.sleep(0)` literally does nothing so get rid of it

File: stab.py [hunk 2]
```diff
@@ -1,106 +1,76 @@
...
-    time.sleep(0.33)
-  sys.stdout.write("\n")
-  sys.stdout.flush()
-
-def laugh(str):
-  for letter in str:
-    sys.stdout.write(letter)
-    sys.stdout.flush()
-    time.sleep(0)
-  sys.stdout.flush()
```
Comment (Iz3as) [inline stab.py:33]: `laugh` wasn't printing new lines, maybe `dur_type` could handle this?

Comment (nathj07) [reply stab.py:33]: I just checked, the only call to laugh (from stab.py) can and should have a new line at the end

**Review by nathj07 — APPROVED** [review]
This is some good refactoring.

🎫 👻

**Pull request merged**